roles/openstack_rabbitmq/tasks/main.yml

```diff
@@ -11,8 +11,13 @@
     stockpile_openstack_rabbitmq_file_descriptors: "{{ rabbitmq_desc.stdout }}"
   when: rabbitmq_desc is succeeded and containers is not defined and rabbitmq_desc.stdout != ""
 
+- name: get rabbitmq bundle name - container
+  shell: "{{ container_cli }} ps | grep rabbit | awk '{print $(NF)}'"
+  become: true
+  register: rabbitmq_name
+
 - name : Get rabbitmq file descriptors - containers
-  shell: "{{ container_cli }} exec rabbitmq rabbitmqctl status | grep total_limit | awk -F',' '{print $2}'| sed 's/.$//'"
+  shell: "{{ container_cli }} exec {{ rabbitmq_name.stdout }} rabbitmqctl status | grep total_limit | awk -F',' '{print $2}'| sed 's/.$//'"
   register: rabbitmq_desc
   ignore_errors: true
   when: containers is defined
```

Comment:
Can you check that return value of rabbitmq_name var to make sure a rabbitmq container exists

Reply:
@asyedham please address this, to see if the registered variable has the "rabbit" string?

So when: containers is defined and 'rabbit' in rabbitmq_name.stdout

for example

Reply:
@smalleni Have updated it. Thanks!

Reply:
@asyedham Where is this updated? I don't see it?
when: containers is defined and 'rabbit' in rabbitmq_name.stdout should be on line 24

Reply:
Here I have added a check at line [(18)](https://github.com/cloud-bulldozer/stockpile/pull/90/files#diff-02cb624c2da2ac471f0f3f78e397038eR18) .